src/connectors/hierarchical-menu/__tests__/connectHierarchicalMenu-test.ts

```diff
@@ -691,6 +691,175 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/hierarchica
         canToggleShowMore: false,
       });
     });
+
+    describe('facetOrdering', () => {
+      const resultsViaFacetOrdering = [
+        {
+          count: 47,
+          data: null,
+          exhaustive: true,
+          isRefined: false,
+          label: 'Outdoor',
+          value: 'Outdoor',
+        },
+        {
+          count: 880,
+          data: [
+            {
+              count: 173,
+              data: null,
+              exhaustive: true,
+              isRefined: false,
+              label: 'Frames & pictures',
+              value: 'Decoration > Frames & pictures',
+            },
+            {
+              count: 193,
+              data: null,
+              exhaustive: true,
+              isRefined: false,
+              label: 'Candle holders & candles',
+              value: 'Decoration > Candle holders & candles',
+            },
+          ],
+          exhaustive: true,
+          isRefined: true,
+          label: 'Decoration',
+          value: 'Decoration',
+        },
+      ];
+      const resultsViaSortBy = [
+        {
+          count: 880,
+          data: [
+            {
+              count: 193,
+              data: null,
+              exhaustive: true,
+              isRefined: false,
+              label: 'Candle holders & candles',
+              value: 'Decoration > Candle holders & candles',
+            },
+            {
+              count: 173,
+              data: null,
+              exhaustive: true,
+              isRefined: false,
+              label: 'Frames & pictures',
+              value: 'Decoration > Frames & pictures',
+            },
+          ],
+          exhaustive: true,
+          isRefined: true,
+          label: 'Decoration',
+          value: 'Decoration',
+        },
+        {
+          count: 47,
+          data: null,
+          exhaustive: true,
+          isRefined: false,
+          label: 'Outdoor',
+          value: 'Outdoor',
+        },
+      ];
+
+      test.each`
+        ordered  | facetOrdering | sortBy          | expected
```

Comment:
I had a hard time understanding what `ordered` meant here. 
I think I know now it means _"the engine returned something in `renderingContent.facetOrdering.values[attribute]`"_. Could we call it something that would make it obivous it's the order dictated by the API response?

I think any of the following would remove the confusion: 
- `responseHasFacetOrdering` 
- `renderingContentFacetOrdering`  
- `responseFacetOrdering`